frontend/src/features/manageProfile/types/manageProfile.d.ts

```diff
@@ -81,10 +80,11 @@ export type VerifyMigratedClientResponse = {
 /**
  * The tabs on the user profile management page.
  */
-export enum BCEID_PROFILE_TABS {
-  COMPANY_INFORMATION = 0,
-  MY_INFORMATION = 1,
-  USER_MANAGEMENT_ORGADMIN = 2,
-  PAYMENT_INFORMATION_CVCLIENT = 2,
-  PAYMENT_INFORMATION_ORGADMIN = 3,
-}
+export const BCEID_PROFILE_TABS = {
+  COMPANY_INFORMATION: 0,
+  MY_INFORMATION: 1,
+  USER_MANAGEMENT_ORGADMIN: 2,
+  //PAYMENT_INFORMATION_CVCLIENT: 2,
```

Comment:
Um it wasn't used anywhere in the code, but the main issue was that it's using the same enum value as USER_MANAGEMENT_ORGADMIN which was confusing (and was giving lint errors)

Reply:
okay, it could probably use a code comment. But the tab number is different for `ORGADMIN` and `CVCLIENT`. That is because, `ORGADMIN` has the user management tab as the third tab which is not applicable to a 'CVCLIENT`

This is why, there are two values:
`PAYMENT_INFORMATION_CVCLIENT` and `PAYMENT_INFORMATION_ORGADMIN`.

Reply:
Is it possible to use PAYMENT_INFORMATION_CVCLIENT: 3, and PAYMENT_INFORMATION_ORGADMIN: 4 instead?

Reply:
I think that messes up the tab count in the ManageProfile component. Which is why I went with the simplest solution which is what makes sense here too.

We could consider disabling the no-duplicate-enums eslint for this specific case.

Reply:
ok